crates/core/component/stake/src/params.rs

```diff
@@ -31,8 +32,8 @@ impl TryFrom<pb::StakeParameters> for StakeParameters {
         Ok(StakeParameters {
             unbonding_epochs: msg.unbonding_epochs,
             active_validator_limit: msg.active_validator_limit,
-            slashing_penalty_downtime: msg.slashing_penalty_downtime,
-            slashing_penalty_misbehavior: msg.slashing_penalty_misbehavior,
+            slashing_penalty_downtime: todo!("convert proto defs"),
+            slashing_penalty_misbehavior: todo!("convert proto defs"),
```

Comment:
do we want to change the parameters to encode a `U128x128`, or should we keep them as some kind of decimal fraction like they are now?